experimental/dds/tree2/src/test/shared-tree/editing.spec.ts

```diff
@@ -290,6 +290,46 @@ describe("Editing", () => {
 			expectJsonTree(tree2, ["B", "A", "C"]);
 		});
 
+		it("can rebase cross-field move over unrelated change", () => {
+			const tree1 = makeTreeFromJson({
+				foo: ["A"],
+				bar: ["B"],
+			});
+			const tree2 = tree1.fork();
+
+			const rootPath = {
+				parent: undefined,
+				parentField: rootFieldKeySymbol,
+				parentIndex: 0,
+			};
+
+			const fooList: UpPath = { parent: rootPath, parentField: brand("foo"), parentIndex: 0 };
+			const barList: UpPath = { parent: rootPath, parentField: brand("bar"), parentIndex: 0 };
+
+			// Change value of A to C
+			tree1.editor.setValue({ parent: fooList, parentField: brand(""), parentIndex: 0 }, "C");
+
+			// Move A after B.
+			tree2.editor.move(
+				{ parent: fooList, field: brand("") },
+				0,
+				1,
+				{ parent: barList, field: brand("") },
+				1,
+			);
+
+			const expectedState: JsonCompatible = {
+				foo: [],
+				bar: ["B", "C"],
+			};
+
+			tree1.merge(tree2);
+			tree2.rebaseOnto(tree1);
+
+			expectJsonTree(tree1, [expectedState]);
+			expectJsonTree(tree2, [expectedState]);
```

Comment:
FYI: you can just do that, but I understand it's nice to be able to breakpoint on a specific check sometimes. As long as you know is option is there I'm happy.
```suggestion
			expectJsonTree([tree1, tree2], [expectedState]);
```